recipe/meta.yaml

```diff
@@ -1,17 +1,15 @@
-{% set version = "1.3" %}
+{% set version = "2.1.0" %}
 
 package:
-  name: flit
+  name: flit-split
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/flit/flit-{{ version }}.tar.gz
-  sha256: 6f6f0fb83c51ffa3a150fa41b5ac118df9ea4a87c2c06dff4ebf9adbe7b52b36
+  sha256: 6c954c3ec4e87a84fac55cca1a1d692a39354affa7c791eb49f6631b71ddbb40
 
 build:
   number: 0
-  script: FLIT_ROOT_INSTALL=1 {{PYTHON}} -m flit install --deps=none  # [not win]
-  script: {{PYTHON}} -m flit install --deps=none  # [win]
   skip: True  # [py2k]
   entry_points:
     - flit = flit:main
```

Comment:
I am not sure, but I would assume that the entrypoints also need to go into the corresponding output section.